README.md

```diff
@@ -1,2 +1,36 @@
 # nma-cn-project
-Project for Neuromatch Academy: Computational Neuroscience
+## Does engagement matter: Do mice see the world differently when they don't care?
```

Comment:
The information conveyed looks good! Let's reword it a little.

---

The following repository contains code for the Project for Neuromatch Academy: Computational Neuroscience. 

Traditionally V1/VISp is considered a simple feature detector. Stimulus representation in A1 is known to adapt to engagement in a go/no-go task. Does stimulus representation in V1 also differ depending on engagement (active) and disengagement (passive)? We investigate this question both cell-level and population-level.

Reply:
Hi @seungjaeryanlee  I was wondering if you could reword it as per your liking?